src/Extension/ExtendableTrait.php

```diff
@@ -525,4 +517,18 @@ public static function extendableCallStatic($name, $params = null)
             $name
         ));
     }
+
+    /**
+     * Gets the class loader
+     *
+     * @return ClassLoader|null
+     */
+    protected function getClassLoader(): ?ClassLoader
```

Comment:
@bennothommo it was named extensionGetClassLoader() to avoid any potential conflicts since this is basically **the** base class for everything in October

Reply:
Ah I see - I'll rename it back.